src/Core/Manifest/VersionProvider.php

```diff
@@ -80,7 +80,7 @@ public function getVersion()
      */
     public function getModuleVersion(string $module): string
     {
-        $key = sprintf('%s-%s', $this->getComposerLockPath(), $module);
+        $key = sprintf('%s_%s', preg_replace("/[^A-Za-z0-9]/", '', $this->getComposerLockPath()), $module);
```

Comment:
`$module` needs to be included in the `preg_replace()` here, as the module name contains slashes (e.g. `silverstripe/framework`)

Reply:
ah yes.. then we might as well remove the sprintf altogether, that's what i just did in the last commit